synapseformation/utils.py

```diff
@@ -48,3 +48,25 @@ def read_config(template_path: str):
 #     def from_yaml(cls, loader, node):
 #         return cls(node.value)
 # # yaml.safe_load('Foo: !Ref bar')
+
+
+def represent_dictionary_order(self, dict_data):
+    """Source: https://stackoverflow.com/a/49048250"""
+    return self.represent_mapping("tag:yaml.org,2002:map", dict_data.items())
```

Comment:
[This](https://yaml.org/type/map.html) is the actual documentation of the mapping, and wow you stumbled upon something really interesting, Bruno! 💯